tests/unittest/executor/test_ray_stub.py

```
@@ -0,0 +1,112 @@
+    @stub.remote(num_gpus=1)
+    class Worker:
+        def run(self):
+            return None
+
+    with pytest.raises(RuntimeError, match="Worker"):
+        Worker()
+
+
+def test_unknown_attribute_raises_and_names_itself():
+    with pytest.raises(RuntimeError, match=r"ray\.init"):
+        stub.init
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)